src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4164,6 +4171,19 @@ private bool HasTopLevelNullabilityConversion(TypeWithAnnotations source, TypeWi
                     break;
 
                 case ConversionKind.ImplicitNullable:
+                    if (trackMembers && node.Kind == BoundKind.Conversion)
```

Comment:
Can this whole check be broken out into a local function? Seems to be duplicated here and for tuple conversions.

Reply:
The check will be simplified in the next PR and removed later.

---
In reply to: [269237714](https://github.com/dotnet/roslyn/pull/34303#discussion_r269237714) [](ancestors = 269237714)